tools/mo/openvino/tools/mo/__init__.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from .convert import convert_model, InputCutInfo, LayoutMap
+from .convert import convert_model
+from openvino.tools.mo.utils.arg_wrappers import InputCutInfo, LayoutMap
```

Comment:
please do not change it

Reply:
if I do not move them , I have round dependencies, I want use these parameters wrappers for provide additional parameters hints based on example input.

Reply:
Also my opinion, it is not good place for storing data types

Reply:
moved back